### DISCUSSION TITLE
dns-query?name=github.com

### DISCUSSION THREAD

**Discussion by ghost** [DNSCrypt/dnscrypt-proxy#2405] [Q&A]
Hi , 

How can I activate json resposne in browser for DoH like below?

https://dns.nextdns.io/dns-query?name=github.com

{"Status":0,"TC":false,"RD":true,"RA":true,"AD":false,"CD":false,"Question":[{"name":"github.com.","type":1}],"Answer":[{"name":"github.com.","type":1,"TTL":300,"data":"140.82.121.3"}],"Additional":[{"name":".","type":41,"TTL":0,"data":"\n;; OPT PSEUDOSECTION:\n; EDNS: version 0; flags:; udp: 1232"}]}

Thanks

**Comment by lifenjoiner** [comment]
It is unsupported. Local DoH follows rfc8484.
